bokeh/_templates/file.html

```diff
@@ -3,11 +3,12 @@
     <head>
         <meta charset="utf-8">
         <title>{{ title if title else "Bokeh Plot" }}</title>
-        {{ plot_resources|indent(8) }}
-
-        {{ plot_script|indent(8) }}
+        {{ bokeh_css }}
     </head>
     <body>
         {{ plot_div|indent(8) }}
+
+        {{ bokeh_js }}
```

Comment:
It would say it's standard practice to put js at the end of the page. I just assumed you weren't because they were all lumped together so you didn't have a choice (as css needs to be in head). I'm not bothered, as people who care about that probably would likely be making their own template anyway.